test/01-smoke.test.js

```diff
@@ -3,16 +3,16 @@ const test = require('brittle')
 const path = require('bare-path')
 const hypercoreid = require('hypercore-id-encoding')
 const Helper = require('./helper')
-const harness = path.join(Helper.localDir, 'test', 'fixtures', 'harness')
+const worker = path.join(Helper.localDir, 'test', 'fixtures', 'worker')
```

Comment:
Lets create another fixture, we can call it `basic worker`, the plan is to replace harness with a much simpler one